config/polypus.yml

```diff
@@ -1,6 +1,6 @@
 http.port: 4000
 
-logger.dir: /var/log/polypus/
+logger.dir: /tmp/log/polypus/
```

Comment:
It should always be the /var/log directory. If you want to change it for developing, you must set an Environment Variable for it.